src/__configuration__/resources/index.tsx

```diff
@@ -123,32 +123,32 @@ export const resources: ResourceBucket[] = [
                 name: '@pxblue/angular-themes',
                 description: 'PX Blue themes for Angular Material',
                 package: '@pxblue/angular-themes',
-                repository: 'themes',
+                repository: 'angular-themes',
                 applies: ['angular'],
-                bugLabels: ['angular'],
+                bugLabels: ['angular-themes'],
```

Comment:
you can remove the bugLabel props entirely from these...since they have dedicated repositories now, all of the bugs will be related to that specific framework.